### DISCUSSION TITLE
Siemens Oven  and new erros

### DISCUSSION THREAD

**Discussion by euDominic** [alexkn/node-red-contrib-homeconnect#47] [Q&A] [answered]
After 3 Months of an working node i get now errors.

Error: Required parameter haId is not provided
![Unbenannt](https://user-images.githubusercontent.com/88461637/128258805-dc3cfe69-129d-4835-9822-c30b8b5acb90.jpg)

Thera are no changes on the System or the nodes.

**Comment by alexkn** [comment, marked as answer]
Looks line Home Connect renamed the haid parameter to haId. (upper case i)
should be fixed in [v0.6.4](https://github.com/alexkn/node-red-contrib-homeconnect/releases/tag/v0.6.4)

**Reply by euDominic** [reply]
Thx for the quick fix :-)

All is now working fine again.
